translations/lua/mudlet-lua.json

```diff
@@ -1,19 +1,62 @@
 {
-  "AdjustableContainer":{
-    "attach": "Attach to:",  
-    "bottom": "bottom",
-    "custom": "Custom:",
-    "lock": "Lock",
-    "lockstyle":"Lockstyle:",
-    "standard":"standard",
-    "light":"light",
-    "full":"full",
-    "border":"border",
-    "top":"top",
-    "left":"left",
-    "right": "right",
-    "load":"Load",
-    "save":"Save",
-    "min_restore": "Min/Restore"
+  "AdjustableContainer.attach": { 
+    "message": "Attach to:",
+    "description":"Attaches your container to a border"
+  },  
+  "AdjustableContainer.bottom": { 
+    "message":"bottom",
+    "description":"is on of the possible borders to attach the containers"
+  },
+  "AdjustableContainer.custom": {
+    "message":"Custom:",
+    "description":"Is a menu item in the right click menu with user/package made custom items in it"
+  },
+  "AdjustableContainer.lock": {
+    "message":"Lock",
+    "description":"Locks the container. Which means it's not movable/resizable anymore and changes some other properties depending on the Lockstyle (like hiding the borders)"
+  },
+  "AdjustableContainer.lockstyle": {
+    "message":"Lockstyle:",
+    "description":"There are different modes/styles to choose which will be the lock behaviour of the container"
+  },
+  "AdjustableContainer.standard": {
+    "message":"standard",
+    "description":"Is one of the Lockstyles. This Lockstyle is the standard setting, with a small margin on top to keep the right click menu usable"
+  },
+  "AdjustableContainer.light": {
+    "message":"light",
+    "description":"Is one of the Lockstyles. Can also be translated as simple. It is only hiding the min/restore labels the borders and margin stays intact"
+  },
+  "AdjustableContainer.full": {
+    "message":"full",
+    "description":"Is one of the Lockstyles. Can also be translated as complete as it's locking the container without any margin left for the right click menu."
+  },
+  "AdjustableContainer.border": { 
+    "message":"border",
+    "description":"is one of the Lockstyles which keeps the borders of the container visible."
+  },
+  "AdjustableContainer.top": {
+    "message":"top",
+    "description":"is one of the possible border the container can be attached to"
+  },
+  "AdjustableContainer.left": {
+    "message":"left",
+    "description":"is one of the possible borders the container can be attached to"
+  },
+  "AdjustableContainer.right": {
+    "message":"right",
+    "description":"one of the borders the container can be attached to"
+  },
+  "AdjustableContainer.load": {
+    "message":"Load",
+    "description":"Loads the previously settings of the container"
```

Comment:
```suggestion
    "description":"Loads the previous settings of the container"
```

Reply:
hm I changed this one already. You are somehow referring to a older version.

the translations (all of them) were more a showcase how it works, I think they should be rechecked/rediscussed on Crowdin.

Reply:
My bad. You are right!